.travis.yml

```diff
@@ -39,6 +39,7 @@ geosx_linux_build: &geosx_linux_build
     # but that would not have solved the problem for the TPLs (we would require extra action to copy them to the mount point).
   - CONTAINER_NAME=geosx_build
     # Now we can build GEOSX.
+  - while sleep 5m; do echo "... still building ..."; done & 
```

Comment:
This means that any job really stuck would last to the build limit (3 hours if I'm correct).
I'd be in favour to making the timeout longer if we can.

Reply:
@TotoGaz Good point.  I'd propose we merge this branch as a working patch, and then come back and revisit other options in a separate PR.  Maybe we can get the `travis_wait` option to work with more effort.  @corbett5 Also suggested maybe migrating from make to ninja for the travis build script, and we could update things like this at the same time.